src/dotnet/commands/dotnet-list/dotnet-list-package/ListPackageReferencesCommand.cs

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Cli;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Tools.Common;
+using Microsoft.DotNet.Tools.NuGet;
+
+namespace Microsoft.DotNet.Tools.List.PackageReferences
+{
+    internal class ListPackageReferencesCommand : CommandBase
+    {
+        //The file or directory passed down by the command
+        private readonly string _fileOrDirectory;
+        private AppliedOption _appliedCommand;
+
+        public ListPackageReferencesCommand(
+            AppliedOption appliedCommand,
+            ParseResult parseResult) : base(parseResult)
+        {
+            if (appliedCommand == null)
+            {
+                throw new ArgumentNullException(nameof(appliedCommand));
+            }
+
+            // Gets the absolute path of the given path
```

Comment:
Nit: while code comments can be extremely valuable, comments like this one should probably be removed in favor of letting the code speak for itself.